Packages/com.unity.inputsystem/InputSystem/Devices/Keyboard.cs

```diff
@@ -757,135 +618,134 @@ protected override void OnRemoved()
                 current = null;
         }
 
+        KeyControl[] m_Keys;
```

Comment:
Use explicit visibility modifiers.

Reply:
Do we have any code style guidelines we follow? I'm asking because Unity's C# conventions specify dropping this (see https://ono.unity3d.com/unity-extra/unity-meta/raw/@/ReferenceSource/CSharp/Assets/CSharpReference.cs : 
`        // |    - Drop redundant access specifiers (leave off 'private' at type scope).` ).

Reply:
I changed it anyways, but it would be nice to clarify what guidelines (if any) we follow?